widgets/runtime-data-source-without-saving-to-config/README.md

```diff
@@ -0,0 +1,15 @@
+# Runtime data source without saving to config
+
+The sample demonstrates how to generate a runtime data source that is not saved to the app config. It is designed to create a data source dynamically based on the provided service URL. It is a simplified version of the 'add data' widget.
+
+## How to use the sample
+
+Clone the [sample repo](https://github.com/esri/arcgis-experience-builder-sdk-resources) and copy this widget's folder (within `widgets`) to the `client/your-extensions/widgets` folder of your Experience Builder installation.
+
+## How it works
+
+No settings are needed.
+
+In `widget.tsx`, once we input a feature layer URL, the widget will fetch the layer definition and then create a data source instance. See the `createDataSource`.
```

Comment:
Could you explain the difference between the preconfigured data source and the runtime-created data source?